atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -300,6 +300,13 @@ def set_group_manifest_info(self, extra, worker_metadatas):
                 pullspec = "{0}/{1}:{2}".format(registry.uri, repo,
                                                 version_release)
                 index['pull'].append(pullspec)
+
+                # Store each digest with according media type
+                index['digests'] = {}
+                for version, digest in manifest_list_digests[0].__dict__.items():
```

Comment:
Let's avoid this and make `ManifestDigest` a bit friendlier to work with, e.g. have it inherit from `UserDict`, have the v1, v2, v2_list etc attributes look up the value from the dict.